addons/compact-nav/addon.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Compact Navbar",
+  "description": "Makes a more compact navbar.",
```

Comment:
Elaborate?

Reply:
Moves the username of the navbar pop-up to inside the pop-up, making a more compact navbar.